cmd/server/cronworkflow/cron-workflow.proto

```diff
@@ -0,0 +1,65 @@
+syntax = "proto3";
+option go_package = "github.com/argoproj/argo/cmd/server/cronworkflow";
+
+import "gogoproto/gogo.proto";
+import "google/api/annotations.proto";
+import "k8s.io/apimachinery/pkg/apis/meta/v1/generated.proto";
+import "github.com/argoproj/argo/pkg/apis/workflow/v1alpha1/generated.proto";
+import "k8s.io/api/core/v1/generated.proto";
+
+package cronworkflow;
+
+message CreateCronWorkflowRequest {
+    string namespace = 1;
+    github.com.argoproj.argo.pkg.apis.workflow.v1alpha1.CronWorkflow cronWorkflow = 2;
+    k8s.io.apimachinery.pkg.apis.meta.v1.CreateOptions createOptions = 3;
+}
+message ListCronWorkflowsRequest {
+    string namespace = 1;
+    k8s.io.apimachinery.pkg.apis.meta.v1.ListOptions listOptions = 2;
+}
+message GetCronWorkflowRequest {
+    string cronWorkflowName = 1;
+    string namespace = 2;
+    k8s.io.apimachinery.pkg.apis.meta.v1.GetOptions getOptions = 3;
+}
+message UpdateCronWorkflowRequest {
+    string cronWorkflowName = 1;
+    string namespace = 2;
+    github.com.argoproj.argo.pkg.apis.workflow.v1alpha1.CronWorkflow cronWorkflow = 3;
+}
+message DeleteCronWorkflowRequest {
+    string cronWorkflowName = 1;
```

Comment:
this should be `name` and not `cronWorkflowName` which is implied by the Server.